kubernetes/kube-state-metrics-cluster-role.yaml

```diff
@@ -15,8 +18,11 @@ rules:
   verbs: ["list", "watch"]
 - apiGroups: ["extensions"]
   resources:
-  - daemonsets
   - deployments
+  verbs: ["list", "watch", "get", "update"]
```

Comment:
That's less about not trusting the code/binary of the addon-resizer, than an attacker being able to modify arbitrary deployments.